costs/README.md

```diff
@@ -136,143 +136,173 @@ cons
 
 ```
 microseconds per operation
- cons_nest-128: 17.45846838358501784683 (-145.227675)
-   cons_nest-1: 17.39702532456590233778 (-158.015124)
-cons_nest-1024: 17.16396016461701279354 (141.669000)
+   cons_nest-1: 0.24570221272216288821 (86.825906)
```

Comment:
Can you add a paragraph above explaining the presentation of results?

I am assuming the number in brackets is the % change from the `1` case, but it could also be the change from the result that comes before ...

Reply:
it's actually the constant factor from the linear regression analysis. This output is copy-pasted from the output of `analyze-benchmark.py`